samples/helloworld/host/Makefile

```diff
@@ -17,4 +17,4 @@ build:
 	$(CC) -o helloworldhost helloworld_u.o host.o $(LDFLAGS)
```

Comment:
Change of `helloworldhost` is missing here

Reply:
Fixed